### DISCUSSION TITLE
Whether to support ssl connection

### DISCUSSION THREAD

**Discussion by HG9558** [ra1u/redis-dart#88] [Q&A]
I'm not sure whether this library supports redis's ssl connection method

**Comment by HG9558** [comment]
void main() async {
  final conn = RedisConnection();
  conn.connectSecure('52.196.177.102', 6379).then((Command command) {
    command.send_object(["AUTH", "default", "djckeid123"]).then((var response) {
      print('main.dart文件 8==============行打印=$response');
      command.send_object(["SET", "key", "1"]).then((var response) => print(response));
    });
  });

  runApp(const MyApp());
}

**Reply by ra1u** [reply]
Can you please explain what is your question and where do you have an issue?
